client/src/components/body/landingpage/citation/professorInformation.js

```diff
@@ -7,14 +7,16 @@ const professorInformation = [
         name: "Daniel Krutz",
         title: "PI",
         bio: "",
-        socials: ["https://danielkrutz.github.io/"]
+        socials: ["https://danielkrutz.github.io/"],
+        network: "sharethis"
     },
     {
         profile_image: samuel_malachowsky_image,
         name: "Samuel Malachowsky",
         title: "Co-PI",
         bio: "",
         socials: ["https://www.se.rit.edu/~samvse/"]
+        
```

Comment:
Going off of the addition to krutz's info, should there be a network field for malachowsky?